assets/ui/EnvironmentInfoScreen.ui

```diff
@@ -0,0 +1,101 @@
+{
+  "type": "EnvironmentInfoScreen",
+  "contents": {
```

Comment:
We use 4-space indentation for JSON. Change this in your IDE settings then autoformat